public/_locales/nl/messages.json

```diff
@@ -0,0 +1,298 @@
+{
+    "appName": {
+        "message": "Niet storen s.v.p."
```

Comment:
The app name translation looks a bit weird to me :thinking:, i tried to translate it online & got `Leid me niet af` [on google translate](https://translate.google.com/?sl=en&tl=nl&text=don%27t%20distract%20me&op=translate), i'm not a native speaker so i don't know which one is really more relevant, WDYT?

Reply:
There are many possible translations, but ‘Niet storen s.v.p.’ is a common phrase to indicate that someone doesn't want to be disturbed or distracted. But ‘Geen afleiding’ would be a good translation as well. Do you want me to change it to that?

Reply:
Finally, that's okay let's go for ‘Niet storen s.v.p.’

Thanks for the updates btw!